### PR TITLE
Always produce a .zip and .tar.gz for the targeting pack

### PR DESCRIPTION
**Pull request by natemcmaster**
Follow-up to #11116

### CODE REVIEW

File: src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj [hunk 1]
```diff
@@ -161,18 +161,17 @@ This package is an internal implementation of the .NET Core SDK and is not meant
...
-      Condition="'$(TargetRuntimeIdentifier)' == 'win-x64'" />
+      DestinationFile="$(ZipArchiveOutputPath)"
+      Overwrite="true" />
+    <!-- Requires Windows 10 version 1803 or newer -->
     <Exec
-      Command="tar -czf $(ArchiveOutputPath) ."
-      WorkingDirectory="$(TargetingPackLayoutRoot)"
-      Condition="'$(TargetRuntimeIdentifier)' == 'linux-x64'" />
-    <Message Importance="High" Text="$(MSBuildProjectName) -> $(ArchiveOutputPath)" />
+      Command="tar -czf $(TarArchiveOutputPath) ."
```
Comment (JunTaoLuo) [inline src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj]: Woah we have tar on windows now?

**Review by JunTaoLuo — APPROVED** [review]

**Review by dougbu — APPROVED** [review]
Looks good.

Side question: Why did you originally made the change at https://github.com/aspnet/AspNetCore/pull/11116/files#diff-7936f6109f5f7b8cb12969aa87896bf4R66 ?

File: src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj (hunk 1 above)
Comment (natemcmaster) [reply src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj]: Yup. curl too https://devblogs.microsoft.com/commandline/tar-and-curl-come-to-windows/

**Comment by natemcmaster** [comment]
> Why did you originally made the change at aspnet/AspNetCore/pull/11116/files#diff-7936f6109f5f7b8cb12969aa87896bf4R66 ?

To decouple build and pack. We still need to build the .dlls for the ref pack because there is rid-specific packaging, like .deb and .rpm files, but otherwise, a rid-specific build doesn't need to produce the rid-less packaging (.nupkg/.zip/.tar.gz)

**Pull request merged**